configure.ac

```diff
@@ -269,12 +269,6 @@ else
   AC_MSG_ERROR(no. Please install intltool package)
 fi
 
-# phantomjs
-
-AC_PATH_PROG([PHANTOMJS], [phantomjs], [], [$srcdir/node_modules/.bin$PATH_SEPARATOR$PATH])
-AM_CONDITIONAL([WITH_PHANTOMJS], [test -n "$PHANTOMJS"])
-AC_SUBST([PHANTOMJS])
```

Comment:
This needs to be replaced with logic to search for chromium, similar to [browser_path() in testlib.py](https://github.com/cockpit-project/cockpit/blob/master/test/common/testlib.py#L1265)

Reply:
Indeed, thanks. Fixed this first to make the environments without chromium installed work.